build/buildcheck.sh

```diff
@@ -28,16 +28,16 @@ fi
 # autoconf 2.68 or newer
 ac_version=`$PHP_AUTOCONF --version 2>/dev/null|head -n 1|sed -e 's/^[^0-9]*//' -e 's/[a-z]* *$//'`
 if test -z "$ac_version"; then
-echo "buildconf: autoconf not found."
-echo "           You need autoconf version 2.68 or newer installed"
-echo "           to build PHP from Git."
+echo "buildconf: autoconf not found." >&2
+echo "           You need autoconf version 2.68 or newer installed" 1>&2
```

Comment:
Question: is there any difference between `>&2` and `1>&2`?

Reply:
```suggestion
echo "           You need autoconf version 2.68 or newer installed" >&2
```

Reply:
Oh, it's my bad.

http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_07_02 says:
> If the number is omitted, the redirection shall refer to standard output (file descriptor 1).

Reply:
@petk I fixed.